pyaedt/modeler/modeler3d.py

```diff
@@ -1153,3 +1154,75 @@ def import_from_openstreet_map(
                     self[obj].transparency = transparency
                     self[obj].color = color
         return scene
+
+    @pyaedt_function_handler
+    def objects_segmentation(self, objects_list, segmentation_thickness=None, segments_number=None):
+        """Get segmentation of an object given the segmentation thickness or number of segments.
+
+        Parameters
+        ----------
+        objects_list : list
+            Objects list to apply the segmentation to.
+            It can either be a list of strings (object names), int (object IDs) or
+            list of :class:`pyaedt.modeler.cad.object3d.Object3d`.
```

Comment:
```suggestion
            It can either be a list of strings (object names), integers (object IDs), or
            a list of :class:`pyaedt.modeler.cad.object3d.Object3d` classes.
```